ktor-server/ktor-server-netty/jvm/src/io/ktor/server/netty/NettyApplicationCallHandler.kt

```diff
@@ -47,6 +54,19 @@ internal class NettyApplicationCallHandler(
         }
     }
 
+    override fun exceptionCaught(ctx: ChannelHandlerContext, cause: Throwable) {
+        when (cause) {
+            is ReadTimeoutException -> {
+                runBlocking {
```

Comment:
I see 2 problems here:
- runBlocking can possibly block netty thread from handling requests if `call.respond` contains blocking logic
- `call.respond` could be executed independently in other thread and `double response` can be thrown. 

Could we try cancelling the coroutine handling call instead, so it respond in `finally` block?